lib/dpl/providers/scalingo.rb

```diff
@@ -8,30 +8,33 @@ class Scalingo < Provider
       required :api_key, [:username, :password]
 
       opt '--app APP', default: :repo_name
-      opt '--api_key KEY', 'scalingo API key', alias: :api_token, deprecated: :api_token
-      opt '--username NAME', 'scalingo username'
-      opt '--password PASS', 'scalingo password'
-      opt '--remote REMOTE', 'Git remote name', default: 'scalingo'
+      opt '--api_key KEY', 'Scalingo API key', alias: :api_token, deprecated: :api_token
+      opt '--username NAME', 'Scalingo username'
+      opt '--password PASS', 'Scalingo password'
+      opt '--region REGION', 'Scalingo region', default: 'agora-fr1', enum: %w(agora-fr1 osc-fr1)
+      opt '--remote REMOTE', 'Git remote name', default: 'scalingo-dpl'
       opt '--branch BRANCH', 'Git branch', default: 'master'
+      opt '--timeout SEC', 'Timeout for Scalingo CLI commands', default: 60
 
       needs :git, :ssh_key
 
-      cmds login_key:   'timeout 2 ./scalingo login --api-token %{api_key} > /dev/null 2>&1',
-           login_creds: 'echo -e \"%{username}\n%{password}\" | timeout 2 ./scalingo login > /dev/null 2>&1',
-           add_key:     './scalingo keys-add dpl_tmp_key %{key}',
-           remove_key:  './scalingo keys-remove dpl_tmp_key',
-           remote_add:  'git remote add %{remote} git@scalingo.com:%{app}.git > /dev/null 2>&1',
+      cmds login_key:   'timeout %{timeout} ./scalingo login --api-token %{api_key} > /dev/null 2>&1',
```

Comment:
```suggestion
      cmds login_key:   'timeout %{timeout} ./scalingo login --api-token %{api_token} > /dev/null',
```

We actually need the error output to assist our customers when an issue arise